packages/sanity-plugin-nacelle-input/src/hooks/useHailFrequency.js

```diff
@@ -0,0 +1,97 @@
+import { useState, useEffect } from 'react'
+
+import useSWR from 'swr'
+import fetch from 'isomorphic-unfetch'
+import config from 'config:@nacelle/sanity-plugin-nacelle-input'
```

Comment:
Looks like some additional ESLint config is needed:

<img width="700" alt="sanity plugin useHailFrequency hook eslint unresolved import" src="https://user-images.githubusercontent.com/5732000/177665238-b3a0af17-2567-479d-b3ef-ee1503291f20.png">

Reply:
Installing the `eslint-sanity-plugin` was throwing some errors due to the top-level lerna `eslint` - so I added a disable comment where needed.  If this isn't ideal let me know.